From foo@baz Tue Apr  9 12:12:43 2002
Date: Tue, 18 Jul 2006 10:59:59 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Driver: multi-threaded hacks

 - Fix "issue" with ata_piix doing multi-threaded boot

Use at your own risk.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/ata/ata_piix.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/ata/ata_piix.c
+++ gregkh-2.6/drivers/ata/ata_piix.c
@@ -1140,7 +1140,7 @@ static int __init piix_init(void)
 	if (rc)
 		return rc;
 
-	in_module_init = 0;
+//	in_module_init = 0;
 
 	DPRINTK("done\n");
 	return 0;
